src/Traits/MockTrait.php

```diff
@@ -247,4 +247,34 @@ protected function getInvocationCount(InvokedCount $matcher): int
             ? $matcher->getInvocationCount()
             : $matcher->numberOfInvocations();
     }
+
+    /**
+     * Mock native function chain. Call chain can be passed as multiple arrays
+     * or already prepared arrays of calls.
+     *
+     * @param string $namespace 
+     * @param array ...$calls
+     */
+    public function mockNativeFunctionChain(string $namespace, array ...$calls): void
+    {
+        $this->mockNativeFunction(
+            namespace: $namespace,
+            callChain: $this->prepareCallChain($calls),
+        );
+    }
```

Comment:
let's just call the `prepareCallChain` method inside the `mockNativeFunction` without implementing a new function